raiden_installer/tokens.py

```diff
@@ -158,11 +158,11 @@ def __init__(self, value: Eth_T):
     },
 )
 
-_SAI = Erc20Token(
-    ticker="SAI",
+_DAI = Erc20Token(
+    ticker="DAI",
     wei_ticker="SEI",
```

Comment:
this should be `DEI` (maybe?)

Reply:
there is no smallest unit of DAI https://www.reddit.com/r/MakerDAO/comments/abadwv/whats_the_standard_name_for_the_smallest_unit_of/